CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - A changelog file.
 
+### Changed
+
+- Require swap-cli user to specify the connection details of the peer they wish to swap with.
+  The swap-cli no longer defaults to connecting to COMIT if connection details are not provided.
+  Connecting buyers to us by default is not consistent with our vision of a decentralised network of sellers.
```

Comment:
This "connecting to COMIT" is a bit vague given that there exists this idea of the COMIT network :D

What do you think of something along the lines of:

```suggestion
  Throughout the public demo phase of this project, the CLI defaulted to an `asb` with peer-ID `12D3KooWCdMKjesXMJz1SiZ7HgotrxuqhQJbP5sgBm2BwP1cqThi` and multi-address `/dns4/xmr-btc-asb.coblox.tech/tcp/9876`.
  Having the defaults made it easy for us to give something to the community that can easily be tested, however it is not aligned with our long-term vision of a decentralised network of sellers.
```

Reply:
I avoided asb because buyers may not know/care what that is. How about:

Throughout the public demo phase of this project, the CLI traded with us by default using with peer-ID `12D3KooWCdMKjesXMJz1SiZ7HgotrxuqhQJbP5sgBm2BwP1cqThi` and multi-address `/dns4/xmr-btc-asb.coblox.tech/tcp/9876`.
  Having the defaults made it easy for us to give something to the community that can easily be tested, however it is not aligned with our long-term vision of a decentralised network of sellers.

Reply:
Works for me :)